router.js

```diff
@@ -34,16 +34,27 @@ var configFileEvent = function (curr, prev) {
 
 var do404 = function (res) {
 	res.writeHead(404);
+	res.header("Content-Type", "application/json; charset=utf-8");
```

Comment:
Should we be providing an application/json header for a 404 with no JSON output? This seems wrong to me.